### ISSUE TITLE
Excel sheet fails to read at all if report row and colun headers is enabled due to invalid excel address conversion 

### ISSUE THREAD

**Issue by michaelDCurran** [nvaccess/nvda#5705]
In a very rare case, Excel can fail to give NVDA an address back in RC (r#c#) format when requested. This completely breaks reading of sheets as a regular expression fails to match.

**Comment by michaelDCurran** [comment]
Bug was seen when testing example in #5558
